src/collections/projects/sample/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: "sample"                
 thumbnail: ./layer5.svg          
-published: false
+published: true
```

Comment:
This might be an accidental publication.

Reply:
I did it intentionally, as the query present in the navbar was failing, as it required a minimum of 2 projects present in the collections folder. @vineethvanga18 will have to change the query and hard-code the project cards in the navbar.